src/rotator_library/providers/kilocode_provider.py

```
@@ -0,0 +1,109 @@
+        """
+        try:
+            response = await client.get(
+                "https://kilo.ai/api/openrouter/models",
+                headers={"Authorization": f"Bearer {api_key}"},
+            )
+            response.raise_for_status()
+            return [
+                f"kilocode/{model['id']}" for model in response.json().get("data", [])
+            ]
+        except httpx.RequestError as e:
+            lib_logger.error(f"Failed to fetch Kilocode models: {e}")
```

Ruff (0.15.4)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)